addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -48,9 +59,10 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
             if (!worklocations.id) {
                 worklocations = worklocations[location] && worklocations[location].find(wl => wl.id == id);
             }
-            if (worklocations) {
+            if (worklocations && worklocations.userId !== user.userId) {
```

Comment:
Maybe add a comment above to say that `worklocations.userId` is the user id of the employee :)